app/utils/StatusHelper.scala

```diff
@@ -59,4 +60,12 @@ object StatusHelper {
     prior.flatMap(_.pensionCharges.flatMap(_.overseasPensionContributions)).isDefined
   }
 
+  def statePensionIsUpdated(pensionsUserData: Option[PensionsUserData]): Boolean = {
+    pensionsUserData.map(_.pensions).map(_.incomeFromPensions).flatMap(_.statePension).flatMap(_.amountPaidQuestion).isDefined
```

Comment:
If these defs are only used in once place is there any point defining it in a helper? This would suggest multiple classes or object need to access to his function and so far i can only see one case ?

Reply:
The existing checks for if a model is updated are added to the StatusHelper utility, hence I followed the same pattern.

Reply:
Yes, this is the place where they can check if state pension is updated within the PensionsUserData model. So anyone who wants to query this can use it.

Reply:
Ok sure, let's leave it for now